app/src/main/java/dadeindustries/game/gc/mechanics/turn/TurnProcessor.java

```diff
@@ -32,10 +35,56 @@ private void computeAiTurns(GlobalGameData globalGameData) {
 		}
 	}
 
-	public boolean processTurn(GlobalGameData globalGameData) {
+	private ArrayList<String> battleSummaries = new ArrayList<String>();
+
+	private Event processBattle(Sector s) {
```

Comment:
What a beautiful opportunity for some javadoc, and/or tests..... ;)

Actually unit and integration testing of the battle code is important as it's something we don't want to accidentally mess up.

Reply:
Also, s can be sector.

Reply:
Shouldn't this be in mechanics.units.UnitActions.java?